ui/src/main/webapp/src/app/registered-application/application-list.component.ts

```diff
@@ -116,8 +116,8 @@ export class ApplicationListComponent extends ExecutionsMonitoringComponent impl
             return false;
 
         this.deleteAppDialog.data = application;
-        this.deleteAppDialog.title = 'Confirm application deletion';
-        this.deleteAppDialog.body = `Do you really want to delete the “${application.title}” application?`;
+        this.deleteAppDialog.title = 'Confirm Application Deletion';
+        this.deleteAppDialog.body = `Are you sure you want to delete “${application.title}”?`;
```

Comment:
Well, but I think this one is really directly related to it. We want to fix texts and be more consistent. Technically we could create separate PR fixing just that, but I don't think it is necessary.